portability-types-common/src/main/java/org/datatransferproject/types/common/models/photos/PhotosContainerResource.java

```diff
@@ -46,6 +48,14 @@ public PhotosContainerResource(
     return photos;
   }
 
+  @Override
+  public Map<String, Integer> getCounts() {
+    return new ImmutableMap.Builder<String, Integer>()
+        .put("photosCount", photos.size())
```

Comment:
Should these be in constants somewhere?

Reply:
Added them as static constants inside PhotosContainerResource